RecoJets/JetProducers/python/PileupJetID_cfi.py

```diff
@@ -1,18 +1,20 @@
 import FWCore.ParameterSet.Config as cms
 
-from RecoJets.JetProducers.PileupJetIDParams_cfi import * 
+from RecoJets.JetProducers.PileupJetIDParams_cfi import *
 
 #_stdalgos_4x = cms.VPSet(full,   cutbased,PhilV1)
 _stdalgos_5x = cms.VPSet(full_5x,cutbased,PhilV1)
 
-#_chsalgos_4x = cms.VPSet(full,   cutbased) 
+#_chsalgos_4x = cms.VPSet(full,   cutbased)
 _chsalgos_5x = cms.VPSet(full_5x_chs,cutbased)
 _chsalgos_74x = cms.VPSet(full_74x_chs,cutbased)
 _chsalgos_76x = cms.VPSet(full_76x_chs,cutbased)
 _chsalgos_80x = cms.VPSet(full_80x_chs,cutbased)
 _chsalgos_81x = cms.VPSet(full_81x_chs,cutbased)
+_chsalgos_94x = cms.VPSet(full_94x_chs,cutbased)
+_chsalgos_102x = cms.VPSet(full_102x_chs,cutbased)
 
-_stdalgos    = _chsalgos_81x
+_stdalgos    = _chsalgos_102x
```

Comment:
this change is supposedly leading to the differences in comparisons, which are not allowed for a production release.
Please revert.
For tests this can be enabled with run2_miniAOD_devel modifier ```run2_miniAOD_devel.toReplaceWith(_stdalgos, _chsalgos_102x)```

Reply:
sorry but I think I am a bit confused. I change this line back and what else do I need to do?

Reply:
please add the line above as @slava77 suggested